server/testserver.go

```diff
@@ -31,7 +33,15 @@ import (
 	"github.com/cockroachdb/cockroach/util/stop"
 )
 
+const (
+	// TestUser is a fixed user used in unittests.
+	// It has a permissions config with read/write permissions
+	// on the "test-user" prefix.
+	TestUser = "test-user"
```

Comment:
you could make this private and have a `TestUser()` method on `TestServer` to avoid another test-related export.